src/sql/base/browser/ui/infoButton/infoButton.ts

```diff
@@ -0,0 +1,183 @@
+
+	constructor(container: HTMLElement, options?: IInfoButtonOptions) {
+		super(container, options);
+		this._container = container;
+
+		{ // Creates the elements
+			this._container.style.display = 'flex';
+			this._container.style.justifyContent = 'space-around';
+
+			this._main = document.createElement('div');
+			DOM.addClass(this._main, 'flexContainer');
+			this._main.style.cursor = 'pointer';
+			this._main.style.backgroundColor = '#FFFFFF';
+			this._main.style.borderRadius = '4px';
+			this._main.style.boxShadow = '0px 1px 4px rgba(0, 0, 0, 0.14)';
+			this._main.style.padding = '10px';
+
+			this._iconContainer = document.createElement('div');
+			this._iconContainer.style.alignItems = 'flex-start';
+			this._iconContainer.style.display = 'flex';
+			this._iconContainer.style.flexFlow = 'column';
+			this._iconContainer.style.paddingRight = '10px';
+
+			this._iconElement = document.createElement('div');
+			DOM.addClass(this._iconElement, 'icon');
+
+			this._textContainer = document.createElement('div');
+			this._textContainer.style.color = '#006ab1';
+			this._textContainer.style.display = 'flex';
+			this._textContainer.style.flexFlow = 'column';
+			this._textContainer.style.justifyContent = 'space-between';
+			this._textContainer.style.padding = '0 0 0 10px';
+			this._textContainer.style.margin = '0px';
+
+			this._pTitle = document.createElement('p');
+			this._pTitle.setAttribute('aria-hidden', 'false');
+			this._pTitle.style.fontSize = '14px';
+			this._pTitle.style.fontWeight = 'bold';
+			this._pTitle.style.lineHeight = '20px';
+			this._pTitle.style.margin = '0px';
+
+			this._pDesc = document.createElement('p');
+			this._pDesc.setAttribute('aria-hidden', 'false');
+			this._pDesc.style.fontSize = '12px';
+			this._pDesc.style.lineHeight = '16px';
+			this._pDesc.style.margin = '0px';
+
+			this._textContainer.appendChild(this._pTitle);
+			this._textContainer.appendChild(this._pDesc);
+
+			this._iconContainer.appendChild(this._iconElement);
+
+			this._main.appendChild(this._iconContainer);
+			this._main.appendChild(this._textContainer);
+			this.element.appendChild(this._main);
+			this.element.style.background = 'none';
+			this.element.style.display = 'inline-block';
+		}
+		this.infoButtonOptions = options;
+	}
+
+	public get textTitle(): string {
+		return this._textTitle;
+	}
+	public set textTitle(value: string) {
+		this._textTitle = value;
+		this._pTitle.innerText = this.textTitle;
+	}
+
+	public get description(): string {
+		return this._description;
+	}
+	public set description(value: string) {
+		this._description = value;
+		this._pDesc.innerText = this.description;
+	}
+
+	public get buttonMaxHeight(): number {
+		return this._buttonMaxHeight;
+	}
+	public set buttonMaxHeight(value: number) {
+		this._buttonMaxHeight = value;
+		this._main.style.maxHeight = this._buttonMaxHeight.toString() + 'px';
+		this._iconContainer.style.height = (this._buttonMaxHeight - 20).toString() + 'px';
```

Comment:
please use template strings whenever you are doing string concatenation in this file, will save you a lot of .toString()

Reply:
Done. Thanks for suggesting this!